docs/docs/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+id: faq
+title: FAQs
+sidebar_label: FAQs
+---
+
+### Why Zephyr™?
```

Comment:
Making these an `##` or `#` should make a nice right sidebar TOC appear, let folks jump to the FAQ they want

Reply:
It appears anyway.  I used ### because # and ## was way too big and I couldn't find an easy grouping.  Happy to change it if we want to change the CSS too.

Reply:
Yeah the ### seems to work on the right sidebar. I think that its size is nice. If we don't want to change the CSS, I think ### is good.

Reply:
Odd. I didn't see it from my phone. Let's leave it as is then